src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -68,6 +70,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
```

Comment:
For case statements maybe add statement to state that no need to do "break;" as return already breaks out of case switch

Reply:
Since has to do with java syntax, I feel that it shouldn't be explained here as its not specific to our code.